test/unit/test_snake.py

```diff
@@ -45,73 +50,75 @@ def setup_dummy_simple(sample_names=('G1234', 'G5678', 'G3671.1_r1', 'G3680-1_4'
         add_to_sample_list(workdir, name, sample_names)
 
     shutil.copy(os.path.join(util.file.get_project_path(), 'pipes', 'Snakefile'), workdir)
-    shutil.copy(os.path.join(util.file.get_project_path(), 'pipes', 'config.yaml'), workdir)
 
-    os.symlink(util.file.get_project_path(), os.path.join(workdir, 'bin'))
+    with open(os.path.join(util.file.get_project_path(), 'pipes', 'config.yaml')) as f:
+        config = yaml.load(f)
+
+    def translate_remote_s3(uri):
+      remote_path = uri[5:]
+      fake_s3_root = os.path.join(util.file.get_project_path(), 'test', 'input', 's3')
```

Comment:
Maybe all of the empty placeholder files should be made dynamically as part of the tests?

Reply:
It's hard to determine what the filenames need to be dynamically. Either need to store a list of filenames to then create at runtime or create empty files - I'm ambivalent about these two.